### PR TITLE
TRIVIAL: Update PR template

### PR DESCRIPTION
**Pull request by petrjanu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/220/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/264/ : SUCCESS in 1m 21s

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -5,5 +5,6 @@ if single-commit, feel free to leave empty)._
 
 # PR checklist
 
-- [ ] Change is described in [CHANGELOG.md](../CHANGELOG.md).
-- [ ] Migration guide (for major update) is written to [CHANGELOG.md](../CHANGELOG.md).
\ No newline at end of file
+- [ ] Change was tested using dev-release in AD and KD (if applicable).
```
Comment (martinmosko) [inline .github/pull_request_template.md]: AD and KD would be better to use full names

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/65/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/266/ : SUCCESS in 31s

**Pull request merged**